.ruby-version

```diff
@@ -0,0 +1 @@
+2.5.0
```

Comment:
Please use 2.6.5

Reply:
Agreed. 2.5.0 was in the gemfile.lock hence keeping it that way initially. Have pushed this to 2.6.5